apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -486,8 +493,46 @@ export class PublishAction extends BaseRushAction {
     }
   }
 
+  private _addNpmPublishHome(): void {
+    // Example: "common\config\rush\.npmrc-publish"
+    const sourceNpmrcPublishPath: string = path.join(this.rushConfiguration.commonRushConfigFolder, '.npmrc-publish');
+
+    // Example: "common\temp\publish-home"
+    const targetNpmrcPublishFolder: string = path.join(this.rushConfiguration.commonTempFolder, 'publish-home');
+
+    // Example: "common\temp\publish-home\.npmrc"
+    this._targetNpmrcPublishPath = path.join(targetNpmrcPublishFolder, '.npmrc');
```

Comment:
Can you do this directly in the constructor so it's obvious that it's set?

Reply:
I was originally going to initialize this in the constructor. But to stay consistent with other rush-lib cli actions I set it in `_addNpmPublishHome` . Currently `ChangeAction`, `InitAction`, `PublishAction`, `VersionAction` are the only actions I see that declare private instance fields that are not `ts-command-line` parameters and all these are not done in the constructor.  I can do it directly in the constructor if you'd like, just noted my reasoning.

Reply:
I think I'd rather it be in the constructor. We should probably go through the other actions and fix those up eventually.

Reply:
Updated with changes. I made `targetNpmrcPublishFolder` private given that `_addSharedNpmConfig()` gets called multiple times and its use in `_addNpmPublishHome`.